installation.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Update package index
+sudo apt update
+
+# Install necessary packages to allow apt to use repositories over HTTPS
+sudo apt install -y apt-transport-https ca-certificates curl software-properties-common
+
+# Add Docker's official GPG key
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg
+
+# Add Docker repository
+echo "deb [signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+
+# Update package index again to include Docker repository
+sudo apt update
+
+# Install Docker
+sudo apt install -y docker-ce
+
+# Start and enable Docker service
+sudo systemctl start docker
+sudo systemctl enable docker
+
+# Add the current user to the "docker" group to run Docker commands without sudo
+sudo usermod -aG docker $USER
+
+# Display Docker version
+docker --version
+
+# Display Docker info
+docker info
+
+# Building the Docker container
+docker build -t your-image-name .
+
+# Running the container
+docker run -p 80:80 --name your-container-name your-image-name
```

Comment:
can we have a better placeholder for the container and image name?

Maybe core-craft and core-craft-vm or something more inclusive to the project.

Reply:
Okay sir... I didn't know what name to give that's why I gave a general name

Reply:
Completely understandable. Not a big issue, but will be expecting a change from someone else before using your script out-of-box.
In terms of best practices, we generally give names related to the project for easy identification and relation. 
Few suggestions would be:

- cc-dev
- core-craft-vm
- core-craft

And no **sir**. Please address either by name or tag by using `@` before the name. :)